core/src/main/scala/cats/data/Validated.scala

```diff
@@ -1121,6 +1129,8 @@ private[data] trait ValidatedFunctions {
    */
   def validNel[E, A](a: A): ValidatedNel[E, A] = Validated.Valid(a)
 
+  def validA[F[_], E, A](a: A): Validated[F[E], A] = Validated.Valid(a)
```

Comment:
this one's a bit weird to me since `Validated` is covariant in both parameters, and also `valid[F[E], A](a)` is really not any longer than `validA[F, E, A](a)` (in fact it is shorter).

Do we need this one?

Reply:
yeah, good point, thanks!